blockstore.go

```diff
@@ -504,6 +504,42 @@ Retry:
 	return err
 }
 
+func (b *Blockstore) DeleteMany(cids []cid.Cid) error {
+	b.oplock.RLock()
+	defer b.oplock.RUnlock()
+
+Retry:
+	err := b.env.Update(func(txn *lmdb.Txn) error {
+		for _, cid := range cids {
+			if err := txn.Del(b.db, cid.Hash(), nil); err != nil {
+				return err
+			}
+		}
```

Comment:
To not shadow the import:

```suggestion
		for _, c := range cids {
			if err := txn.Del(b.db, c.Hash(), nil); err != nil {
				return err
			}
		}
```